api/app/settings/common.py

```diff
@@ -579,31 +581,11 @@
         },
         "loggers": {
             "": {"level": LOG_LEVEL, "handlers": ["console"]},
-            # Not sure why the following loggers are necessary, but it doesn't seem to
-            # write log messages for e.g. features.workflows.core.models without adding
-            # them explicitly.
-            # TODO: move all apps to a parent 'apps' directory and configure the logger
-            #  for that dir
-            "features": {
-                "level": LOG_LEVEL,
-                "handlers": ["console"],
-                "propagate": False,
-            },
             "task_processor": {
                 "level": LOG_LEVEL,
                 "handlers": ["console"],
                 "propagate": False,
             },
```

Comment:
@khvn26 good catch - the logging wasn't working correctly for the task processor. I'm not entirely sure why but I assume it's related to it having been moved to it's own package. I've re-added the configuration for it here in the meantime.